verilog/tools/kythe/kythe_facts_extractor.cc

```diff
@@ -307,6 +379,62 @@ VName KytheFactsExtractor::ExtractClassInstances(
   return class_instance_vname;
 }
 
+VName KytheFactsExtractor::ExtractPackageImport(
+    const IndexingFactNode& import_fact_node) {
+  const auto& anchors = import_fact_node.Value().Anchors();
+  const Anchor& package_name = anchors[0];
```

Comment:
Is it guaranteed that there will be at least one anchor? Is a size check needed?

Reply:
For package imports the facts tree node would contain two anchors in case of import pkg::myVar one for pkg and the other for myVar and in case of import pkg::* we will have only one anchor for pkg.